src/js/content_scripts/injector.js

```diff
@@ -105,6 +105,11 @@ const constructParams = () => {
   return params;
 };
 
+const openLiveTL = () => {
+  window.top.location =
+    chrome.runtime.getURL(`watch.html?${constructParams()}`);
+};
```

Comment:
Couldn't we just export this in `utils.js` and import it whenever its needed instead of doing postMessage?

Reply:
We'd also need to extract `constructParams()`, but extracting should be fine.